README.md

```diff
@@ -4,3 +4,5 @@ This has been updated
 Changes have been made on the new branch
 
 
+More changes were made during review.
+a second local branch is being used.
```

Comment:
this looks good. it is a better way of communicating by going directly to what has been changed.